packages/samples/test/output/testserver/multiple-inheritance/openapi.json

```diff
@@ -309,7 +309,7 @@
               }
             }
           },
-          "default": {
+          "4XX": {
```

Comment:
This feels a bit wrong to me that we are passing this "unexpected error" as 4xx

Reply:
I will update the response description.